projects/js-packages/publicize-components/src/components/form/index.tsx

```diff
@@ -61,7 +65,9 @@ export default function PublicizeForm() {
 					<PanelRow>
 						<ConnectionsList />
 					</PanelRow>
-					{ feature_flags.useEditorPreview && isPublicizeEnabled ? <SocialPostModal /> : null }
+					{ feature_flags.useEditorPreview && isPublicizeEnabled && ! isPostPublished ? (
```

Comment:
I am still not sure about this. We still use the feature flag here but we have also removed the old reshare button. Shouldn't we either not use feature flag here or restore that resharing button if the feature flag is not active?

Reply:
Ah OK. Thanks